### ISSUE TITLE
Appendicular spasticity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7649]
**Preferred term label:**
Appendicular spasticity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620001 
PMID: 34385670

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
